packages/react-scripts/scripts/init.js

```diff
@@ -124,52 +124,19 @@ module.exports = function(
   // Install react and react-dom for backward compatibility with old CRA cli
   // which doesn't install react and react-dom along with react-scripts
   // or template is presetend (via --internal-testing-template)
-  if (!isReactInstalled(appPackage) || template) {
```

Comment:
This diff is atrocious, but if [we compare upstream/master to origin/fork](https://github.com/facebook/create-react-app/compare/master...Skyscanner:fork?expand=1#diff-0ae46a2383dedcac5f5e369f5ca5f169) - you'll see a bunch of stuff we do to add backpack dependencies an shift stuff around.

All i am doing is deleting that and making use of the `.template.dependencies.json` file to install out custom dependencies.